index.js

```diff
@@ -61,6 +61,7 @@ module.exports = {
         var dotFolders            = this.readConfig('dotFolders');
         var serverSideEncryption  = this.readConfig('serverSideEncryption');
         var batchSize             = this.readConfig('batchSize');
+        var defaultMimeType       = this.readConfig('defaultMimeType');
```

Comment:
@achambers sure thing. Just added it now. 
Interestingly though, `application/octet-stream` is already set to the default by the mime module if no default is supplied, but I guess its better be to more explicit.